polling_stations/templates/base.html

```diff
@@ -62,22 +62,36 @@
 {% endblock content %}
 {% block mailing_list %}{% endblock mailing_list %}
 {% block footer_menu %}
-    <div class="ds-cluster-center">
-        <p class="ds-h4">{% trans "Where Do I Vote?"%}</p>
-        <ul class="ds-padded" style="list-style:none;">
-            <li><a href="{% url 'home' %}">{% trans "Home" %}</a></li>
-            <li><a href="https://democracyclub.org.uk/projects/polling-stations/">{% trans "About Where Do I Vote?" %}</a></li>
-            <li><a href="https://democracyclub.org.uk/about/">{% trans "About Democracy Club" %}</a></li>
-            <li><a href="https://democracyclub.org.uk/privacy/">{% trans "Privacy" %}</a></li>
-            <li><a href="https://democracyclub.org.uk/contact">{% trans "Contact Us" %}</a></li>
-            <li><a href="https://democracyclub.org.uk/blog/">{% trans "Blog" %}</a></li>
-            <li><a href="https://twitter.com/democlub/">{% trans "Twitter" context "link to Democracy Club Twitter page" %}</a></li>
-            <li><a href="https://facebook.com/DemocracyClub/">{% trans "Facebook" context "link to Democracy Club Facebook page" %}</a></li>
-            <li><a href="https://github.com/DemocracyClub/">{% trans "GitHub" context "link to Democracy Club GitHub page" %}</a></li>
-        </ul>
-    </div>
+    {% block footer_links %}
```

Comment:
In this PR, I have just copied the links we already had under 3 headings. So its the same links laid out differently

I think if we want to actually review the footer content (for example: do we want to link people to mastodon, bluesky, etc these days?) lets do that in another PR.